examples/evalboards/TMC5130/six_point_ramp_demo.py

```diff
@@ -54,55 +54,70 @@ def speed_rotation2step(x): return x * micro_steps_per_mechanical_revolution
+    eval_board.write_register_field(mc.FIELD.IRUN,2)        # IRUM      | set the standstill current
+    eval_board.write_register_field(mc.FIELD.IHOLD,2)       # IHOLD     | set Motor run current
+    eval_board.write_register_field(mc.FIELD.SG_STOP, 0)    # SG_STOP   | resets the StallGuard2 - Stop
+    eval_board.write_register_field(mc.FIELD.SGT, 0)        # SGT       | turns StallGuard2 off
+    def speed_step2rotation(x):
+        return x / (micro_steps_per_mechanical_revolution / pow(2 , M_Res))
+    def speed_rotation2step(x):
+        return x * (micro_steps_per_mechanical_revolution / pow(2 , M_Res))
+
+    # Set - ramp - parameters
+    #########################                                               # Name     |  Mode   |           Task
+    eval_board.write_register(mc.REG.A1,  round(8000 / pow(2, M_Res)))      # A1       | 6 piont | initial acceleration between VSTART and V1
+    eval_board.write_register(mc.REG.AMAX, round(800 / pow(2, M_Res)))      # AMAX     | trapez. | accelaration in the end
+    eval_board.write_register(mc.REG.V1, round(100000 / pow(2,M_Res)))      # V1       |         | threshold for the  first acceleration phase
+    eval_board.write_register(mc.REG.D1,round(8000 / pow(2, M_Res)))        # D1       | 6 piont | de/acceleration in the  end / start
+    eval_board.write_register(mc.REG.DMAX, round(800 / pow(2, M_Res)))      # DMAX     | trapez. | initial deacceleration
+    eval_board.write_register(mc.REG.VSTART, 0 )                            # VSTART   |         | Motor start velocity
+    eval_board.write_register(mc.REG.VSTOP, 10)                             # VSTOP    |         | Motor stop velocity threshold
 
     print("Rotating...")
-    motor.rotate(-v_max)
+    eval_board.write_register_field(mc.FIELD.RAMPMODE, 2) # aktivate velocity mode in negative direction
     time.sleep(5)
 
     print("Stopping...")
-    motor.stop()
+    eval_board.write_register_field(mc.FIELD.VMAX, 0)       # set speed to 0
+    eval_board.write_register_field(mc.FIELD.RAMPMODE, 2)   # apply changes
 
-    while motor.actual_velocity != 0:
+    while eval_board.read_register_field(mc.FIELD.VACTUAL) != 0:
         time.sleep(0.1)
 
     print("Moving back to 0...")
-    motor.move_to(0, -v_max)
+    Traget_position = 0
+    eval_board.write_register_field(mc.FIELD.VMAX, v_max)                   # set max speed
+    eval_board.write_register_field(mc.FIELD.XTARGET, Traget_position)      # set traget position to 0
+    eval_board.write_register_field(mc.FIELD.RAMPMODE, 0)                   # aktivate position mode
 
     # Wait until position 0 is reached
-    i = 0                         # Sample count
-    T_s = 0.005                   #Sampling Time [seconds]  = Resolution of the plot ; Rage (0.5 ; 0.005)
+    i = 0         # Sample count
+    T_s = 1E-3 # Sampling Time [seconds] = Resolution of the plot ; Rage (0.5 ; 0.005) for M_res = 0 else smaler is possible
```

Comment:
write as 0.001 to be more clear, the comment is a bit misleading, maybe better use 0.005